deployment/Makefile

```diff
@@ -0,0 +1,11 @@
+ANSIBLE = ansible-playbook -i hosts.yml
+
+ifndef PIAF_MACHINE_USER
+	PIAF_MACHINE_USER:=$$USER
```

Comment:
why the `$$` ? 

`PIAF_MACHINE_USER:=$USER` could not work ?

Reply:
It wont work, because $USER is a Makefile variable (which does not exists), hence the double $